llvm/include/llvm/Analysis/ScalarEvolution.h

```diff
@@ -1316,6 +1316,11 @@ class ScalarEvolution {
 
     LoopGuards(ScalarEvolution &SE) : SE(SE) {}
 
+    static LoopGuards
+    collectFromBlock(ScalarEvolution &SE, ScalarEvolution::LoopGuards &Guards,
```

Comment:
If the passed in guards are the. same as the returned ones, is there a need to return them?

Could you also add a doc-comment for the function?

Reply:
yes, no need to return, updated to void and added comment